docs/en/setup/backend/backend-receivers.md

```diff
@@ -128,3 +129,20 @@ receiver-oc:
     gRPCHost: ${SW_OC_RECEIVER_GRPC_HOST:0.0.0.0}
     gRPCPort: ${SW_OC_RECEIVER_GRPC_PORT:55678}
 ```
+
+## Kafka receiver
+
+Kafka Receiver supports to receive `TraceSegment`, `JVMMetrics` and others of Agent reported as gRPC Receiver.
+Currently, it has to connect to the whole topic or single partition of topic when OAP is in cluster mode.
+And we have to configure **Zookeeper** as the OAP cluster management tool.
+
+**Suggestion and recommend**, creating the Kafka topics by self before OAP Server starting. Although OAP server
```

Comment:
What are the topics, and what are they for? And which are the recommend commands with parameters? We need a very clear doc about this.

Reply:
`Suggestion and recommend` are not the same. Please make this part clear. Also, you have codes to create topic automatically, what they are going to do, also should be declared clearly here.